23.0.0.9-beta staging (#3304)
OpenLiberty/blogs: posts/2050-09-05-23.0.0.9-beta.adoc

---
layout: post
title: "Spring Boot 3 support and Jakarta Data preview in Open Liberty 23.0.0.9-beta"
# Do NOT change the categories section
categories: blog
author_picture: https://avatars3.githubusercontent.com/mbroz2
author_github: https://github.com/mbroz2
seo-title: Spring Boot 3 support and Jakarta Data preview in Open Liberty 23.0.0.9-beta - OpenLiberty.io

> seo-description: "The Open Liberty 23.0.0.9-beta release continues to make progress on two features that have been made available via previous betas: Spring Boot 3 support and the Jakarta Data preview."

Reviewer: ```suggestion
seo-description: "The Open Liberty 23.0.0.9-beta release continues progress on two features that were introduced in previous betas: Spring Boot 3 support and the Jakarta Data preview."
```

> blog_description: "The Open Liberty 23.0.0.9-beta release continues to make progress on two features that have been made available via previous betas: Spring Boot 3 support and the Jakarta Data preview."

Reviewer: ```suggestion
blog_description: "The Open Liberty 23.0.0.9-beta release continues progress on two features that were introduced in previous betas: Spring Boot 3 support and the Jakarta Data preview."
```

open-graph-image: https://openliberty.io/img/twitter_card.jpg
open-graph-image-alt: Open Liberty Logo
---
= Spring Boot 3 support and Jakarta Data preview in Open Liberty 23.0.0.9-beta
Michal Broz <https://github.com/mbroz2>
:imagesdir: /
:url-prefix:
:url-about: /
//Blank line here is necessary before starting the body of the post.

> The Open Liberty 23.0.0.9-beta release continues to make progress on two features that have been made available via previous betas: Spring Boot 3 support and the Jakarta Data preview.

Reviewer: ```suggestion
The Open Liberty 23.0.0.9-beta release continues progress on two features that were introduced in previous betas: Spring Boot 3 support and the Jakarta Data preview.
```
